### PR TITLE
scss for home page and its components🥴

### PR DESCRIPTION
**Pull request by ashuvssut**
## Issue that this pull request solves

 Closes: #417 

------------------
_--I have participated in DWoC SWoC and MWoC_

### CODE REVIEW

**Comment by ashuvssut** [comment]
⚠⚠⚠

Please do not merge this PR now. Merge #426 first

I realized that the imports for buttons need to be implemented the right way. The directory of button.jsx has changed in #426. I will update this PR wrt #426.

for now review #426 and suggest changes/corrections (if any). after that gets merged successfully...I will update this one accordingly

**Comment by Kajol-Kumari** [comment]
@ashuvssut please resolve the conflict here and #425

**Comment by ashuvssut** [comment]
pushing changes within few minutes (10mins)

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**
